RecoTracker/MkFitCore/interface/MkBuilder.h

```diff
@@ -0,0 +1,186 @@
+#ifndef RecoTracker_MkFitCore_interface_MkBuilder_h
+#define RecoTracker_MkFitCore_interface_MkBuilder_h
+
+#include <vector>
+#include <map>
+
+//------------------------------------------------------------------------------
+
+#include "RecoTracker/MkFitCore/interface/IterationConfig.h"
+#include "RecoTracker/MkFitCore/interface/Track.h"
+#include "RecoTracker/MkFitCore/interface/HitStructures.h"
+
+#include <functional>
+#include <mutex>
+
+namespace mkfit {
+
+  class CandCloner;
+  class LayerInfo;
+  class MkFinder;
+  class MkFitter;
+  class TrackerInfo;
+
+  class Event;
+
+  //==============================================================================
+  // MkJob
+  //==============================================================================
+
+  class MkJob {
+  public:
+    const TrackerInfo &m_trk_info;
+    // Config &config; // If we want to get rid of namespace / global config
+    const IterationConfig &m_iter_config;
+    const EventOfHits &m_event_of_hits;
+
+    const IterationMaskIfcBase *m_iter_mask_ifc = nullptr;
+
+    int num_regions() const { return m_iter_config.m_n_regions; }
+    const auto regions_begin() const { return m_iter_config.m_region_order.begin(); }
+    const auto regions_end() const { return m_iter_config.m_region_order.end(); }
+
+    const auto &steering_params(int i) { return m_iter_config.m_steering_params[i]; }
+
+    const auto &params() const { return m_iter_config.m_params; }
+    const auto &params_bks() const { return m_iter_config.m_backward_params; }
+
+    int max_max_cands() const { return std::max(params().maxCandsPerSeed, params_bks().maxCandsPerSeed); }
+
+    const std::vector<bool> *get_mask_for_layer(int layer) {
+      return m_iter_mask_ifc ? m_iter_mask_ifc->get_mask_for_layer(layer) : nullptr;
+    }
+  };
+
+  //==============================================================================
+  // MkBuilder
+  //==============================================================================
+
+  class MkBuilder {
+  protected:
+    void fit_one_seed_set(TrackVec &simtracks, int itrack, int end, MkFitter *mkfttr, const bool is_brl[]);
+
+    MkJob *m_job = nullptr;
+
+    // MIMI -- To be removed. Used by seed processing / validation that has yet to be moved.
```

Comment:
please move into an issue or remove.

Reply:
Addressed in Round 2 follow-up PR trackreco/cmssw#76 commit https://github.com/trackreco/cmssw/pull/76/commits/508d5f2980f1fda48a4444273235a55d33da9314 : the comment was rephrased.